catalog.bom

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright &copy; 2013 Cloudsoft Corporation Limited
+#
+# Licensed under the Cloudsoft EULA v1.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.cloudsoftcorp.com/cloudsoft-developer-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+brooklyn.catalog:
+  items:
+  - id: entity-with-new-volume-customizer
+    version: "0.6.0-SNAPSHOT" # BROOKLYN_BLOCKSTORE_VERSION
+    name: New Volume Customizer
+    description: Attaches additional disks in provisioning time
+    itemType: template
+    item:
+      services:
+      - type: org.apache.brooklyn.entity.software.base.EmptySoftwareProcess
```

Comment:
A wish I have DeepDsl evaluation to be able to make configurable catalog items like the one here with simpler syntax.
Simpler example (not working):

```yaml
  - type: org.apache.brooklyn.entity.software.base.EmptySoftwareProcess
    id: entity-with-disks
    brooklyn.config:
      firstSuffix: h
      secondSuffix: g
      provisioning.properties:
        customizers:
          - '$brooklyn:object':
              type: >-
                io.brooklyn.blockstore.brooklyn-blockstore:brooklyn.location.blockstore.NewVolumeCustomizer
              brooklyn.config:
                volumes:
                  - blockDevice:
                      sizeInGb: 3
                      deviceSuffix: $brooklyn:config("firstSuffix")
                      deleteOnTermination: true
                      tags:
                        brooklyn: br-example-test-1
                    filesystem:
                      mountPoint: $brooklyn:formatString("/mnt/brooklyn/%s",$brooklyn:config("firstSuffix"))
                      filesystemType: ext3
                  - blockDevice:
                      sizeInGb: 3
                      deviceSuffix: g
                      deleteOnTermination: true
                      tags:
                        brooklyn: br-example-test-1
                    filesystem:
                      mountPoint: $brooklyn:formatString("/mnt/brooklyn/%s",$brooklyn:config("secondSuffix"))
                      filesystemType: ext3
```